lambdas/check-coi/src/main/java/uk/gov/di/ipv/core/checkcoi/CheckCoiHandler.java

```diff
@@ -97,31 +99,26 @@ public Map<String, Object> handleRequest(ProcessRequest request, Context context
                                             .stream())
                             .toList();
 
-            var coiSubjourneyType = ipvSession.getCoiSubjourneyType();
+            var checkType = RequestHelper.getCoiCheckType(request);
```

Comment:
(@MikeCollingwood this will conflict with your PR, but hopefully it'll be obvious how to resolve - whoever merges second!)